workflows/test_build_rock.yml

```diff
@@ -0,0 +1,45 @@
+name: Test build rock
+
+on:
+  workflow_call:
+    inputs:
+      rockspecs:
+        required: true
+        type: string
+
+jobs:
+  build:
+    name: Test build rockspec
+    strategy:
+      fail-fast: false
+      matrix:
+        # Upstream Issue: https://github.com/leafo/gh-actions-luarocks/issues/8
+        luaVersion: [ "5.4", "5.3", "5.2", "5.1" ] #, "luajit", "luajit-openresty"]
+        # luarocksVersion: [ "3.1.3" ] # , "2.4.2"]
+        rockspec: ${{ fromJSON(inputs.rockspecs) }}
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup ‘lua’
+        uses: leafo/gh-actions-lua@v9
+        with:
+          luaVersion: ${{ matrix.luaVersion }}
+      - name: Setup ‘luarocks’
+        uses: leafo/gh-actions-luarocks@v4
+        # with:
+          # luarocksVersion: ${{ matrix.luarocksVersion }}
+      - name: Confirm dev rockspec(s) build
+        if: ${{ !startsWith(matrix.rockspec, 'rockspecs/') }}
+        run: |
+          args="--lua-version ${{ matrix.luaVersion }}"
+          luarocks make --pack-binary-rock $args -- ${{ matrix.rockspec }}
```

Comment:
This seems weird. Why not
```
luarocks make --pack-binary-rock --lua-version ${{ matrix.luaVersion }} -- ${{ matrix.rockspec }}
```

Reply:
Because other iterations of this had a *lot* more code there and lines were unmanageable. It is probably possible to simplify now that I cleaned up some of the rest of the logic. On the other hand this is an easy jumping off point for passing an argument to the workflow to get added here, so I might leave it for now.

Reply:
why the `--pack-binary-rock` option?

Reply:
So that it doesn't try to install it to the CI runner, only build and collect all the things that would be installed. Installing to the system runs into permission issues on some platforms and the point of the test is to make sure it builds, not make sure `luarocks` can install things to the environment.

Reply:
ok. Thx. Maybe some comments regarding choices like this could be helpful.